src/test/java/seedu/address/storage/XmlAdaptedPersonTest.java

```diff
@@ -11,23 +11,22 @@
 import org.junit.Test;
 
 import seedu.address.commons.exceptions.IllegalValueException;
-import seedu.address.model.person.Address;
-import seedu.address.model.person.Email;
-import seedu.address.model.person.Name;
-import seedu.address.model.person.Phone;
+import seedu.address.model.person.*;
```

Comment:
Change the settings of your Intellij, don't allow it to auto change the import to have the asterisk